src/libraries/System.ServiceProcess.ServiceController/src/System/ServiceProcess/ServiceController.cs

```diff
@@ -927,7 +927,12 @@ public void Stop()
         /// <param name="stopDependentServices">
         /// <c>true</c> to stop all running dependent services together with the service; <c>false</c> to stop only the service.
         /// </param>
-        public unsafe void Stop(bool stopDependentServices)
+#if NETCOREAPP3_1_OR_GREATER
+        public
+#else
+        private
+#endif
```

Comment:
This is probably a bit hacky, but it seemed much better than duplicating the implementation of `Stop()` to me.